src/Components/Authorization/Mobile/RegisterForm.tsx

```diff
@@ -40,58 +68,73 @@ export const MobileRegisterForm: FormComponentType = props => {
         </div>
       )}
     </Step>,
-    <Step>
+    <Step validationSchema={Validators.password}>
       {({
         wizard,
         form: { errors, touched, values, handleChange, handleBlur },
       }) => (
         <div>
           <Input
             block
-            error={touched.password && errors.password}
+            error={errors.password}
             name="password"
             placeholder="Password"
             type="password"
             value={values.password}
             onChange={handleChange}
             onBlur={handleBlur}
+            quick
           />
         </div>
       )}
     </Step>,
-    <Step>
+    <Step validationSchema={Validators.name}>
       {({
         wizard,
         form: { errors, touched, values, handleChange, handleBlur },
       }) => (
         <div>
           <Input
             block
-            error={touched.name && errors.name}
+            error={errors.name}
             name="name"
             placeholder="Name"
             type="text"
             value={values.name}
             onChange={handleChange}
             onBlur={handleBlur}
+            quick
           />
         </div>
       )}
     </Step>,
   ]
   return (
-    <Wizard steps={steps} validationSchema={Validators}>
+    <Wizard steps={steps}>
       {context => {
-        const { wizard } = context
+        const { form, wizard } = context
         const { currentStep } = wizard
         return (
           <Container>
+            <ProgressIndicator
+              percentComplete={
+                wizard.isLastStep
```

Comment:
I think we could abstract that logic into the wizard and expose it as `wizard.progressPercentage: number`.
which is a value from 0 to 1